README.md

```diff
@@ -88,6 +90,9 @@ Potential expression formats with examples:
 | `gdal-calc` | string | A [`gdal_calc.py`](https://gdal.org/programs/gdal_calc.html) expression based on numpy syntax. | `A*logical_or(A<=177,A>=185)` |
 | `openeo`    | object | [openEO process](https://openeo.org/documentation/1.0/developers/api/reference.html#section/Processes) | [Example](https://raw.githubusercontent.com/Open-EO/openeo-processes/1.2.0/normalized_difference.json) |
 | `rio-calc`  | string | A [rio-calc](https://rasterio.readthedocs.io/en/latest/topics/calc.html) (RasterIO) expression | `(b4-b1)/(b4+b1)` |
+| `tiling` | object | key value pairs describing how to tile the data, for example `{ "colormap": "viridis", "rescale": [0,1], "resampling": "nearest" }`. See [`collection-modis-landcover-houston.json`](./examples/collection-modis-landcover-houston.json) for an example. |
```

Comment:
Wouldn't rescale be solved by providing statistics (i.e. bands can have min/max statistics).

Reply:
depends if you want to rescale using custom min/max instead of dataset min/max